src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts

```diff
@@ -37,15 +41,28 @@ export class DotPageStateService {
         return lockUnlock$.mergeMap(() =>
             pageMode$.map(
                 (updatedPage: DotRenderedPage) =>
-                    new DotRenderedPageState(
-                        this.loginService.auth.loginAsUser || this.loginService.auth.user,
-                        updatedPage,
-                        state.mode
-                    )
+                    new DotRenderedPageState(this.loginService.auth.loginAsUser || this.loginService.auth.user, updatedPage, state.mode)
             )
         );
     }
 
+    /**
+     * Get page state
+     *
+     * @param {string} url
+     * @returns {Observable<DotRenderedPageState>}
+     * @memberof DotPageStateService
+     */
+    reload(url: string): Observable<DotRenderedPageState> {
```

Comment:
Why do we need to return anything here?

Reply:
to refresh the `pageState` on the right sidenavbar

Reply:
You can achieve the same thing by subscribing `reload$` we don't need to subscribe to 2 different sources of the same data.